charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml

```diff
@@ -2,11 +2,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ .Release.Name }}-clusterrole
+  name: argo-rollouts-clusterrole
   labels:
-    app.kubernetes.io/component: {{ .Values.controller.component }}
-    app.kubernetes.io/name: {{ .Release.Name }}-clusterrole
-    app.kubernetes.io/part-of: {{ .Release.Name }}
+    app.kubernetes.io/component: rollouts-controller
```

Comment:
Why are you hardcoding these labels?  That seems undesireable.

Reply:
Ah, good catch. I'll revert that change and keep the templated values.